package.json

```diff
@@ -29,6 +29,8 @@
     "bluebird": "^2.5.2",
     "body-parser": "^1.10.0",
     "busboy": "^0.2.9",
+    "c3": "^0.4.7",
+    "d3": "^3.5.3",
```

Comment:
you're adding d3 but not requiring it anywhere? i assume it's a dependency of c3 then... but not in c3's package.json?